pkg/synchronizer/track.go

```diff
@@ -131,8 +137,9 @@ func (t *TrackSynchronizer) GetPTS(pkt *rtp.Packet) (time.Duration, error) {
 	estimatedPTS := time.Since(t.startTime)
 	if pts < t.lastPTS || !t.acceptable(pts-estimatedPTS) {
 		newStartRTP := ts - t.toRTP(estimatedPTS)
-		t.logger.Infow("correcting PTS",
-			"TS", ts,
+		t.logger.Infow(
+			"correcting PTS",
+			"currentTS", ts,
```

Comment:
changing this as `ts` is a reserved field in DataDog. This particular case was okay as it is case sensitive and this still got displayed, but don't want to risk collision.